### PR TITLE
Changes to Diagrams According to the Comments

### LINKED ISSUE #103: Issue with AllocatorSequence
The return arrow should start exactly at the bottom edge of the activation bar (not slightly higher). 
image

### LINKED ISSUE #105: Issue with AllocatorClass
It will be better if you include an empty apartment for attributes in ```Allocator``` class for clarity. 
image
Please write association label on the middle of the line instead of at the end. 
image

### PR DESCRIPTION
**Pull request by ThisisXXZ**
close #105 
close #103 

Main changes include:
* ``Allocator`` class diagram.
  * include an empty apartment for attributes
  * move the association label on the middle of the line
* ``AllocatorCommand`` sequence diagram.
  * unbold ``Allocator`` class
  * extend activation bars to align them with textbook style
  * align the return arrow with the end of the activation bar of ``UI`` class
* ``ViewQuotaCommand`` sequence diagram.
  * remove ``spotsLeft`` attribute name
* Rename PPP file name.

### CODE REVIEW

File: docs/UML_Diagrams/AllocatorSequence.drawio.svg
Comment (isaacsaw25) [inline docs/UML_Diagrams/AllocatorSequence.drawio.svg:1]: My bad on this, but the `boolean` return value of parser --> findoursep should contain a colon, i.e. `:boolean` 😰 😰

**Review by isaacsaw25 — CHANGES_REQUESTED** [review]
Apologies for my previous confusion, the return type should contain a colon. 🙏

File: docs/UML_Diagrams/AllocatorSequence.drawio.svg
Comment (ThisisXXZ) [reply docs/UML_Diagrams/AllocatorSequence.drawio.svg:1]: No problem! Have added back

**Review by isaacsaw25 — APPROVED** [review]
Thanks for the prompt change 🙏

**Pull request merged**